tests/integration/cmd_push_test.go

```diff
@@ -313,7 +316,7 @@ var _ = Describe("odo push command tests", func() {
 				cmpName,
 				appName,
 				project,
-				[]string{"stat", "/tmp/src/views/index.html"},
+				[]string{"stat", filepath.Join(dir, "src/views/index.html")},
```

Comment:
filepath.Join(dir, "src/views/index.html") -> filepath.Join(dir, "src", "views", "index.html")

Follow the same pattern for all other occurrences